ARE/components/processor.adjustmentcurve/src/main/java/eu/asterics/component/processor/adjustmentcurve/AdjustmentCurveInstance.java

```diff
@@ -354,7 +404,29 @@ public Object setRuntimePropertyValue(String propertyName, Object newValue) {
             }
             return oldValue;
         }
-
+        if ("operationMode".equalsIgnoreCase(propertyName)) {
+            final int oldValue = propOperationMode;
+            propOperationMode = Integer.parseInt(newValue.toString());
```

Comment:
in theory newValue could be null.

But this would also be a problem for the other properties

Reply:
same question as in https://github.com/asterics/AsTeRICS/pull/294#discussion_r274802960